core/util/filters.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""
+This file contains Qudi methods for data filtering.
+
+Qudi is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Qudi is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Qudi. If not, see <http://www.gnu.org/licenses/>.
+
+Copyright (c) the Qudi Developers. See the COPYRIGHT.txt file at the
+top-level directory of this distribution and at <https://github.com/Ulm-IQO/qudi/>
+"""
+
+import numpy as np
+from scipy.ndimage import minimum_filter1d, maximum_filter1d
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+def scan_blink_correction(image, axis=1):
```

Comment:
Please document what this does.

Reply:
In general most methods and newly introduced functionality lacks documentation. I will add more anyways.

Reply:
Done